### PR TITLE
[bug]: resolve segfault on dynamic plugin error

### PR DESCRIPTION
**Pull request by darosior**

### CODE REVIEW

File: lightningd/plugin_control.c [hunk 1]
```diff
@@ -34,9 +34,13 @@ static struct command_result *plugin_dynamic_list_plugins(struct command *cmd)
 static struct command_result *
 plugin_dynamic_error(struct dynamic_plugin *dp, const char *error)
 {
-	plugin_kill(dp->plugin, "%s: %s", dp->plugin->cmd, error);
+	if (dp->plugin)
+		plugin_kill(dp->plugin, "%s: %s", dp->plugin->cmd, error);
+	else
+		log_info(dp->cmd->ld->log, error);
```
Comment (rustyrussell) [inline lightningd/plugin_control.c]: Needs to be `log_info(dp->cmd->ld->log, "%s", error);`

**Comment by darosior** [comment]
hmm.. Uncovered some more bugs..

**Comment by darosior** [comment]
Ok should be corrected now, I arbitrarily reduced the time out while at it (which then allows to restore `test_plugin_slowinit`). Sorry for this just before the rc ..

**Comment by niftynei** [comment]
ACK https://github.com/ElementsProject/lightning/pull/3151/commits/e7fac1c8d0baeffb1c45320b77863540feca7a08

**Pull request merged**